### ISSUE TITLE
seq array being saved as bag array

### ISSUE THREAD

**Issue by MetadataDeluxe** [adobe-dmeservices/custom-metadata#1]
tested on version 1.1.0

XMP properties that should be seq arrays are being written as bag arrays. There are many seq properties in IPTC Extension, and other popular schemas. Would it be possible to add an "XMP type" field to the View Editor?

**Issue: When a file has no existing values in a seq array, i.e., no dc:creator exists, if a value is created, it is saved as a bag**

Example:
First dc:creator added to a file:

```
<dc:creator>
	<rdf:Bag>
	   <rdf:li>CREATOR</rdf:li>
	</rdf:Bag>
</dc:creator>
```

**Issue: When a file has an existing seq array, i.e. dc:creator already exists, if an additional value is added, the property is saved as a bag**

Example:
Exisiting values created with default Adobe metadata panel
```
<dc:creator>
	<rdf:Seq>
		<rdf:li>CREATOR1</rdf:li>
		<rdf:li>CREATOR2</rdf:li>
	</rdf:Seq>
</dc:creator>
```

After adding another entry "CREATOR3" with the Custom Metadata Panel 1.1.0
```
<dc:creator>
	<rdf:Bag>
	   <rdf:li>CREATOR1</rdf:li>
	   <rdf:li>CREATOR2</rdf:li>
	   <rdf:li>CREATOR3</rdf:li>
	</rdf:Bag>
</dc:creator>
```

**Comment by jameslockman** [comment]
Thanks fo reporting this. We're already scoping this work and should have it in the next release.

**Comment by jameslockman** [comment]
This is solved and will be supported in the next release. We're buttoning that up now and should have it available here by the end of next week. Release via Exchange will take a little longer, as reviews are on hold due to MAX.

**Comment by jameslockman** [comment]
We closed this issue with release 1.2.0. You can now select Bag or Seq when creating a multi-text, multi-dropdown or tags field.
We also introduced item reordering for multi-text fields.
